web/src/modules/feedback/components/FeedbackBox.js

```diff
@@ -66,8 +58,6 @@ export const FeedbackBox = ({
 }: PropsType) => (
   <StyledFeedbackBox>
     <ModalHeader t={t} closeFeedbackModal={closeFeedbackModal} title={t('feedback')} />
-    <Description>{t('feedbackType')}</Description>
-    <Dropdown items={feedbackOptions} selectedItem={selectedFeedbackOption} onOptionChanged={onFeedbackOptionChanged} />
```

Comment:
Do we still use this dropdown somewhere or could we remove it? Does it use a library which could be removed?

Reply:
Removed with unused libraries.